src/IoC.Providers/Cogworks.AzureSearch.IoC.Umbraco/tests/UnitTests/Cogworks.AzureSearch.IoC.Umbraco.UnitTests/Cogworks.AzureSearch.IoC.Umbraco.UnitTests.csproj

```diff
@@ -0,0 +1,52 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <IsPackable>false</IsPackable>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <TargetFrameworks>net6.0;net5.0</TargetFrameworks>
```

Comment:
I think we should include net7 too

Reply:
This will be on separate PR, for now only moving projects from repos to this. After that, there will be a few PR's with workflow updates, linting updates, packages updates (including framework updates)

Reply:
Basically I am moving those repos:
 * https://github.com/thecogworks/Cogworks.AzureSearch.IoC.Umbraco
 * https://github.com/thecogworks/Cogworks.AzureSearch.IoC.Microsoft
 * https://github.com/thecogworks/Cogworks.AzureSearch.IoC.Autofac
 * https://github.com/thecogworks/Cogworks.AzureSearch.IoC.LightInject